modules/integration/src/test/scala/scala/cli/integration/RunTestDefinitions.scala

```diff
@@ -1024,4 +1024,29 @@ abstract class RunTestDefinitions(val scalaVersionOpt: Option[String])
         expect(output.contains(exceptionMsg))
       }
     }
+
+  test("should respect using directives in a .scala file with the shebang line") {
```

Comment:
In this case, we should add unit tests. It should be enough to add a unit test and verify that the using directives are parsed correctly.

Reply:
should be fixed